src/components/CalculatorFilters.vue

```diff
@@ -2,10 +2,18 @@
     <div>
         <Accordion>
             <AccordionTab v-for="group in calculatorStore.filterProperties" :key="group.id" :header="group.label">
+                <div v-if="group.id === 'nist'" class="checkbox-group">
```

Comment:
The "select all" checkboxes can be simplified and made more consistent by just having one of the form:
```vue
<Checkbox
  :modelValue="isSelected(group.id)"
  :disabled="isSelected(group.id)"
  :binary="true"
  :input-id="`select_all_${group.id}`"
  @change="select(group.id)" 
></Checkbox>
<label :for="`select_all_${group.id}`" class="my-auto">All {{ group.label }}</label>
```
Every `filter`, instead of being an `Array`, would be a `Set` and the following heuristic would apply:
  - If the `Set` is empty, then no filters are applied (all items are effectively selected).
  - If the `Set` is _not_ empty, then only the filters contained in the `Set` are applied.

#### Select All Checkboxes:
- `isSelected()`, given a group id, checks if the associated `Set` is empty. If it is, return `true`, `false` otherwise.
- `select()`, given a group id, clears the associated `Set`.

#### Regular Checkboxes (below the Select All checkboxes):

```vue
<Checkbox
  :modelValue="isSelected(group.id, option.name)"
  @change="select(group.id, option.name)"
  class="my-auto"
/>
```
- `isSelected()` (the same function as above) when given BOTH a group id AND a name, checks if the name is contained in the associated `Set`. If it is, return `true`, `false` otherwise.
- `select()` (the same function as above) when given BOTH a group id AND a name, adds the name to the `Set`. If the `Set` and available options are equal in size, clear the Set instead (all items are selected).